docs/docsite/rst/community/development_process.rst

```diff
@@ -143,7 +143,13 @@ We do not merge every PR. Here are some tips for making your PR useful, attracti
 Changelogs
 ----------
 
-Changelogs help users and developers keep up with changes to Ansible. Ansible builds a changelog for each release from fragments. You **must** add a changelog fragment to any PR that changes functionality or fixes a bug in ansible-core. You do not have to add a changelog fragment for PRs that add new modules and plugins, because our tooling does that for you automatically.
+Changelogs help users and developers keep up with changes to Ansible and Ansible collections. Ansible and many collections build changelogs for each release from fragments. For ansible-core and collections using this model, you **must** add a changelog fragment to any PR that changes functionality or fixes a bug. You do not have to add a changelog fragment for PRs that add new modules and plugins, because our tooling does that for you automatically.
```

Comment:
@felixfontein your thoughts ^

Reply:
Can you adjust your suggestion so that it uses `;`/`.` for the list? Then I'd merge it.

About the second part, I'd rather use `Changelogs help users and developers keep up with changes to ansible-core and Ansible collections.` (the correct spelling is `ansible-core` with lower-case `a`). Also not all collections are part of Ansible :)

Reply:
Done, PTAL. If it's not what you meant, feel free to change it yourself, np

Reply:
After that, could you please rephrase the second and the third sentences: 1) it starts with Ansible and 2) there are too many repetitions in the first paragraph